src/applications/financial-status-report/components/debtsAndCopays/AvailableDebtsAndCopays.jsx

```diff
@@ -47,6 +50,19 @@ const AvailableDebtsAndCopays = ({ formContext }) => {
       setSelectionError(
         formContext.submitted && !data.selectedDebtsAndCopays?.length,
       );
+
+      const elidgeible = isEligibleForStreamlined(data);
```

Comment:
eligible 🤣

Reply:
Spelling is hard 😅 commenting as I address these locally, and will resolve convo after next commit.